zerver/openapi/zulip.yaml

```diff
@@ -15327,6 +15354,33 @@ paths:
                           - 2 = Nobody
 
                           **Changes**: New in Zulip 3.0 (feature level 1).
+                      realm_direct_message_initiator_group:
+                        type: integer
+                        allOf:
+                          - description: |
+                              The ID of the [user group](/api/get-user-groups) whose members are
+                              allowed to initiate direct message thread.
+
+                              Previously, access to send direct messages was controlled by the
+                              private_message_policy realm setting, which supported values of
+                              1 (enabled) and 2 (disabled).
```

Comment:
This paragraph should be a second sentence in the **Changes** section.

We should also define terms a bit more.

> The ID of the user group whose members have permission to start a new direction message conversation involving other non-bot users. Users who are outside this group and attempt to send the first direct message to a given collection of recipient users will receive an error, unless all other recipients are bots.


(Actually, I'm not sure what the bot semantics should be -- is that clear in the spec?).

Reply:
I have revised this. I think the bot semantics are clear. I have also added `...or the sender` at the end.